client/my-sites/subscribers/components/grow-your-audience/grow-your-audience.tsx

```diff
@@ -0,0 +1,74 @@
+import { Card, CardBody, Icon } from '@wordpress/components';
+import { chartBar, people, trendingUp } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'calypso/state';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import './styles.scss';
+
+type GrowYourAudienceCardProps = {
+	icon: JSX.Element;
+	text: string;
+	title: string;
+	url: string;
+};
+
+const GrowYourAudienceCard = ( { icon, text, title, url }: GrowYourAudienceCardProps ) => {
+	const translate = useTranslate();
+
+	return (
+		<Card className="grow-your-audience__card" size="small">
+			<CardBody className="grow-your-audience__card-body">
+				<h3 className="grow-your-audience__card-title">
+					<Icon className="grow-your-audience__card-icon" icon={ icon } size={ 20 } />
+					{ title }
+				</h3>
+				<span className="grow-your-audience__card-text">{ text }</span>
```

Comment:
Comparing the design with the rendered cards, the line-heights don't match:

| Design | browser |
|--------|--------|
| ![Markup on 2023-06-13 at 09:22:43](https://github.com/Automattic/wp-calypso/assets/25105483/e261e71f-695f-405a-8f09-18db2390e072) | ![Markup on 2023-06-13 at 09:23:09](https://github.com/Automattic/wp-calypso/assets/25105483/10572e53-f313-490b-976f-8772290a211a) | 

But, the CSS numbers (e.g. `line-height` and `font-sizes`) are correct.

→ Replacing `<span />` with `<p />` fixes the issue.

```suggestion
				<p className="grow-your-audience__card-text">{ text }</p>
```